test2code-common/src/commonMain/kotlin/com/epam/drill/plugins/test2code/common/api/Model.kt

```diff
@@ -20,22 +20,38 @@ import kotlinx.serialization.*
 
 const val DEFAULT_TEST_NAME = "unspecified"
 
+/**
+ * Ast metadata about the file or the class containing methods
+ * @param path the path to class or file
+ * @param name the name of class or file
+ * @param methods the list of methods
+ */
 @Serializable
 data class AstEntity(
     val path: String,
     val name: String,
-    val methods: List<AstMethod>,
+    var methods: List<AstMethod>,
 )
 
+/**
+ * Ast metadata about the method of class or file
+ * @param name the name of the method
+ * @param params the list of parameters of the method
+ * @param returnType the method return type
+ * @param count todo
+ * @param probes the probe indices of the method
+ * @param checksum the checksum of the method body
+ */
 @Serializable
 data class AstMethod(
     val name: String,
     val params: List<String>,
     val returnType: String,
-    val count: Int = 0,
     val probes: List<Int> = emptyList(),
     val checksum: String = "",
-)
+) {
+    val count: Int = probes.size
```

Comment:
reasoning for adding `val count` (calling `probes.size` is already simple enough? 🤔 )

Reply:
bc backward-compatibility with older code